contracts/RToken.sol

```diff
@@ -1,44 +1,39 @@
 pragma solidity ^0.5.8;
 pragma experimental ABIEncoderV2;
 
+import {Storage} from "./Storage.sol";
+import {Structs} from "./Storage.sol";
```

Comment:
I'm a bit anal on this,  please make it `import {Structs} from "./Structs.sol"` <-- or I still insist changing it to RTokenStructs :)

Reply:
Done